balboa-agent/src/main/java/com/socrata/balboa/agent/MetricConsumer.java

```diff
@@ -71,6 +71,24 @@ public void run() {
         }
     }
 
+    /**
+     * Closing the Metric Consumer is effectively a delegation method that allows the Metric Consumers
+     * internal resources to close and clean up (If necessary).
+     *
+     * @throws IOException
```

Comment:
If you aren't going to explain anything extra about this, you might as well delete this line. I can see this info in the type signature.